src/apis/token-service.ts

```diff
@@ -23,24 +23,29 @@ function getTokenMetadataURL(chainId: string, tokenAddress: string) {
   return `${END_POINT}/token/${chainId}?address=${tokenAddress}`;
 }
 
+const tenSecondsInMilliseconds = 10_000;
```

Comment:
[nit] Maybe these should be uppercase so they're not confused for local variables?

Reply:
Done in 34b7b7ad9bdb4456e6040821dc7a4c91f1349106